lib/Net/Async/Blockchain/ETH.pm

```diff
@@ -559,4 +576,33 @@ sub _to_string {
     return substr($packed_response, 0, $size);
 }
 
+=head2 get_numeric_from_hex
+
+Check if the given hexadecimal is numeric and returns the based big number
+
+=over 4
+
+=item * C<hex> hexadecimal response from the blockchain client
+
+=back
+
+Returns a L<Math::BigFloat> value based on the given hexadecimal if it is numeric
+not numeric will return undef
+
+=cut
+
+sub get_numeric_from_hex {
+    my ($self, $hex) = @_;
+
+    my $check_string = $self->_to_string($hex);
+    return undef if $check_string;
+
+    # numeric responses should have at least 64 characters
```

Comment:
Comment says "at least 64", code checks for exactly 66.

Reply:
I was mentioning this in the next line, I did a change in the comment hope it is more clear now